pkg/report/report.go

```diff
@@ -37,166 +37,166 @@ type Result struct {
 
 func (res *Result) Duration() time.Duration { return res.End.Sub(res.Start) }
 
-type report struct {
+// Report contains test results.
+type Report struct {
 	results   chan Result
-	precision string
+	Precision string
 
-	avgTotal float64
-	fastest  float64
-	slowest  float64
-	average  float64
-	stddev   float64
-	rps      float64
-	total    time.Duration
+	AvgTotal float64
```

Comment:
wrap all these in a `Stats` struct and have `Run() <- chan Stats` so there's no way to access them while report is actively collecting data? That way `report` doesn't have to be promoted to a public struct so less code churn